Library/Formula/rfcdiff.rb

```diff
@@ -1,14 +1,42 @@
-require 'formula'
-
 class Rfcdiff < Formula
-  homepage 'http://tools.ietf.org/tools/rfcdiff/'
-  url 'http://tools.ietf.org/tools/rfcdiff/rfcdiff-1.41.tgz'
-  sha1 '02bcd3bf6078caec001bf677530c97893edf34a1'
+  homepage "https://tools.ietf.org/tools/rfcdiff/"
+  url "https://tools.ietf.org/tools/rfcdiff/rfcdiff-1.41.tgz"
+  sha1 "02bcd3bf6078caec001bf677530c97893edf34a1"
+
+  depends_on "wdiff"
+  depends_on "gawk" => :recommended
+  depends_on "txt2man" => :build
 
-  depends_on 'wdiff'
-  depends_on 'gawk' => :recommended
+  resource "rfc42" do
+    url "https://tools.ietf.org/rfc/rfc42.txt"
+    sha1 "5d6a73c63f2d69deba3849bcb7f7c8be158a016c"
+  end
+
+  resource "rfc43" do
+    url "https://tools.ietf.org/rfc/rfc43.txt"
+    sha1 "9f4c5dd7b20c4c62c31217b19d93a41054376bd8"
+  end
 
   def install
-    bin.install "rfcdiff"
+    # replace hard-coded paths
+    inreplace "Makefile.common", "/usr/share/man/man1", man1
+    # don"t use sudo
+    inreplace "Makefile.common", "sudo install", "install"
```

Comment:
@mikemcquaid Yep, I sent a patch to upstream and included it in a `patch do` block.